storage/innobase/include/srv0srv.h

```diff
@@ -117,76 +117,76 @@ struct srv_stats_t
 	ulint_ctr_1_t		buf_pool_reads;
 
 	/** Number of bytes saved by page compression */
-	ulint_ctr_64_t          page_compression_saved;
+	ulint_ctr_n_t          page_compression_saved;
 	/* Number of index pages written */
-	ulint_ctr_64_t          index_pages_written;
+	ulint_ctr_n_t          index_pages_written;
 	/* Number of non index pages written */
-	ulint_ctr_64_t          non_index_pages_written;
+	ulint_ctr_n_t          non_index_pages_written;
```

Comment:
Where is this being incremented? I only see reads of this variable (which should always be 0).

Similarly, are all these counters actually being used somewhere? We could clean up many of them.

Reply:
> Thank you, the changes look OK to me. I will wait for some performance test results for 10.5 to determine where to push this.
> 

Marko, I have uploaded the results for 10.5 in jira ticket with a quick summary as part of the reply?

Reply:
I’ll file a separate ticket to remove the unused counters.